src/screens/TrendingScreen.js

```diff
@@ -0,0 +1,248 @@
+import React, { Component } from "react";
+import {
+  View,
+  Text,
+  StyleSheet,
+  SafeAreaView,
+  Dimensions,
+  FlatList,
+  TouchableOpacity,
+  ActivityIndicator,
+} from "react-native";
+import { SearchBar } from "react-native-elements";
+import Colors from "../constants/Colors";
+import { Image } from "react-native-elements";
+import { BallIndicator } from "react-native-indicators";
+import { Button } from "react-native-paper";
+import dataTrending from "../data/dataTrending";
+
+const { width } = Dimensions.get("window");
+const mostViewsData = dataTrending.sort((a, b) => (b.views - a.views));
+
+export default class TrendingScreen extends Component {
+  state = {
+    loading: false,
+    data: [],
+    searchData: dataTrending,
+    length: 10,
+    error: false,
+    refreshing: false,
+    filteredData: [],
+  };
+
+  inputSearch = "";
+
+  componentDidMount() {
+    this.setState({ loading: true }, () => this.makeRemoteRequest());
+  }
+
+  makeRemoteRequest = () => {
+    const { length } = this.state;
+    const newData = mostViewsData;
```

Comment:
I am not sure what you tried to assign the variable for `newData`. Can you elaborate that on line 67 and 20? We want to avoid any confusion about variables.